### ISSUE TITLE
Update of netdisco-mibs doesn't work

### ISSUE THREAD

**Issue by gmourani** [netdisco/netdisco#700]
Hello,

I've downloaded "netdisco-mibs-4.018" to update the one running on my system which is 3.60. I've also downloaded the corresponding "oui.txt" file and executed the following commands after copying the contents of "netdisco-mibs-4.018" to "netdisco-mibs".

su - netdisco -c "~/bin/netdisco-deploy ./oui.txt"
service netdisco-backend restart
service netdisco-web restart

## Expected Behavior
Under "System Information" on the WUI I expect to see SNMP::Info 4.018 but still see 3.60 !

## Your Environment

* Netdisco version used: 2.39.30
* SNMP::Info version used: 4.018
* Perl version used: 5.10.1
* Linux version used: CentOS 6.9 64bits.

**Comment by zoeloe** [comment]
IMHO SNMP::Info is version 3.70 now. (https://metacpan.org/pod/SNMP::Info)
SNMP::Info Uses the mibs contained in the package available at (https://github.com/netdisco/netdisco-mibs/releases/latest/)
These mibs are normally downloaded if you choose to do that with the ~/bin/netdisco-deploy script
you can also download the OUI data here.
The mibs are version 4.018 separate from SNMP:Info which has version 3.70
You can install SNMP::Info with ~/bin/localenv cpanm --notest SNMP::Info if needed.
Somebody correct me if I'm wrong.

**Comment by inphobia** [comment]
snmp::info & netdisco-mibs versions are different things. we make sure if a new version of snmp::info is released the current mibs at that time will support all features.
but for the most part the netdisco-mibs package is backwards compatible with old snmp::info versions and we don't report it's version anywhere else as in the included README file of the mibs.

**Comment by gmourani** [comment]
OK, thanks to clarify this point to me. SO usually if I want to have the latest MIBs files with Netdisco, I just have to download latest netdisco-mibs tar file and overwrite to the existing one on my server, right ? No additional commands to run to make it used ?

**Comment by inphobia** [comment]
indeed, but an even easier option is to the `netdisco-deploy` command:
https://metacpan.org/pod/distribution/App-Netdisco/bin/netdisco-deploy

just run it, answer `y` to "everything is in place" and "download and update the mibs", and `n` (the default) to all other questions. this will only update your mibs, it won't touch anything else.

```
testdisc@linux002:~> netdisco-deploy
This is the Netdisco 2 deployment script.

Before we continue, the following prerequisites must be in place:
 * Database added to PostgreSQL for Netdisco
 * User added to PostgreSQL with rights to the Netdisco Database
 * "~/environments/deployment.yml" file configured with Database dsn/user/pass
 * A full backup of any existing Netdisco database data
 * Internet access (for OUIs and MIBs)

If you are upgrading Netdisco 2 read the release notes:
https://github.com/netdisco/netdisco/wiki/Release-Notes
There you will find required and incompatible changes
which are not covered by this script.

You will be asked to confirm all changes to your system.

So, is all of the above in place? [y/N]: y

Would you like to deploy the database schema? [y/N]: n

Download and update vendor MAC prefixes (OUI data)? [y/N]: n 

Download and update MIB files? [y/N]: y
MIBs update complete.
```

can't find a way to add more styling in a codeblock to highlight the answers, but i guess it's kinda obvious.

**Comment by gmourani** [comment]
Thanks for the info. I'm trying to make this MIBs update because Netdisco can't see my FortiSwitches devices, so I'm hoping that by updating the MIBs files, maybe new entries will exist for this kind of devices but look like still no support for FortiSwitch. I think I've to open a new issue or a pull requests in the netdisco-mibs GIT site !

**Comment by inphobia** [comment]
aahh, saw that ticket too: netdisco/netdisco-mibs#14

i've downloaded the mib, but it will need snmp::info support most likely. now, the mib doesn't contain a lot of data, which is both good and bad. good is that it's most likely easy to support, bad is that if they don't support standard mibs some features will not be available. but since i have no fortiswitch devices it's hard to say.

we always accept new snmp::info modules if you're willing to write one. if that's not your thing we also have some info on getting the needed data to us: https://github.com/netdisco/snmp-info/wiki/Simulating-Agents#get-snmp-datadump (snmpsim/snmprec method highly preferred)

i however am already backed up as it is. i'm willing to take a look at it but it might take a month or 2 to get to it...

**Comment by gmourani** [comment]
Good, I'll try to use it to generate something to you and will sent it.

**Comment by inphobia** [comment]
in the mean while have added the fortiswitch mib and update a few other easy ones: netdisco/netdisco-mibs#115

do note that netdisco-deploy downloads the latest mibs release, not the current git master branch. so these will be in netdisco-mibs 4.019 and later.

**Comment by gmourani** [comment]
Ok, I've the snmpwalk.out file ready. Where can I upload it ?
